Modules/_stat.c

```diff
@@ -226,10 +230,22 @@ typedef unsigned short mode_t;
 #  define UF_COMPRESSED 0x00000020
 #endif
 
+#ifndef UF_TRACKED
+#  define UF_TRACKED 0x00000040
+#endif
+
+#ifndef UF_DATAVAULT
+#  define UF_DATAVAULT 0x00000080
+#endif
+
 #ifndef UF_HIDDEN
 #  define UF_HIDDEN 0x00008000
 #endif
 
+#ifndef SF_SETTABLE
+#  define SF_ARCHIVED 0xffff0000
+#endif
```

Comment:
Why is this needed?

Reply:
Sigh, I've check the commit a couple of times and still missed that I forgot to update the name here.